src/constants/pools.ts

```diff
@@ -54,6 +54,12 @@ export type NamedPools = {
   veBAL: string;
 };
 
+type DeprecatedDetails = {
+  newPool?: string;
+  suggestedPools?: string[];
+  joinsDisabled?: boolean;
+};
```

Comment:
You may as well add an explicit boolean for disabling staking too.
```suggestion
type DeprecatedDetails = {
  newPool?: string;
  suggestedPools?: string[];
  joinsDisabled?: boolean;
  stakingDisabled?: boolean;
};
```

Reply:
ah sure, thx. added disabling staking as well